config/settings/development.yml

```diff
@@ -1,6 +1,6 @@
 host_inventory_url: http://inventory-web:8081
-remediations_url: https://ci.cloud.redhat.com
-rbac_url: https://ci.cloud.redhat.com
+remediations_url: https://ci.console.redhat.com
+rbac_url: https://ci.console.redhat.com
```

Comment:
I'm not sure if CI/QA will be available on the new location.

Reply:
That's a bit scary :scream: :laughing: